src/osmium_config.json

```diff
@@ -145,5 +145,36 @@
         "railway=station",
         "railway=subway",
         "railway=tram"
-    ]
+    ],
+    "road_tags": {
+        "highway=bridleway": ["highway", "ridleway"],
```

Comment:
Just noticed the tag typo... wonder if there's any way to check them all without doing it manually...